assets/js/components/DashboardEntityApp.js

```diff
@@ -55,9 +55,13 @@ import { Cell, Grid, Row } from '../material-components';
 import PageHeader from './PageHeader';
 import Layout from './layout/Layout';
 import { CORE_WIDGETS } from '../googlesitekit/widgets/datastore/constants';
+import { useHasScrolledEffect } from '../hooks/useHasScrolledEffect';
+import ScrollEffect from './ScrollEffect';
 const { useSelect } = Data;
 
 function DashboardEntityApp() {
+	useHasScrolledEffect();
```

Comment:
This should only be used in the `ScrollEffect` component, as you have in the `DashboardMainApp` below 👍

Reply:
🤦 Forgot to remove that.